packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -205,8 +205,12 @@ const extractContentForAllyMessage = (props: ChatMessageContentProps): string =>
 };
 
 const generateLiveMessage = (props: ChatMessageContentProps): string => {
-  const liveAuthor = _formatString(props.strings.liveAuthorIntro, { author: `${props.message.senderDisplayName}` });
-
+  const liveAuthorIntro = props.message.editedOn
```

Comment:
Why do we `toLowerCase` on only one of these paths?

Reply:
`Edited` string is added for edited message announcement. See the functionality in:
```
return `${props.message.editedOn ? props.strings.editedTag : ''} ${
    props.message.mine ? '' : liveAuthor
  } ${extractContentForAllyMessage(props)} `;
```

Reply:
I'm curious then if this is the right place to lowercase it? We're doing that to make it fit with English word position, correct? Does this apply across all languages?

Reply:
Will check o nthe strings with the accessibility team and update the PR